cwms-aaa-client/src/main/java/mil/army/usace/hec/cwms/aaa/client/CwmsLoginController.java

```diff
@@ -37,11 +37,12 @@
 
 public final class CwmsLoginController {
 
+    static final String JSESSIONID = "JSESSIONID";
```

Comment:
JSESSIONID and JSESSIONIDSSO were needed outside the class.  I'm curious - if they are final String why package protected and not public?  I

Reply:
So far they aren't needed publicly. I'm keeping everything locked down that I can change it whenever I want in the future. Also, there is no need for calling API's to know that the Tomcat auth is using CWMS AAA with a JSESSIONIDSSO cookie, if this changes to JWT (which was proposed before) then I won't be leaking the cookie implementation detail.

The flaw in my argument here is that I need the calling API to pass in a preferences node and provide the CookieJarSupplier which means some implementation details are escaping, but at least WHAT cookie is being used isn't public? I dunno....

Reply:
hmm ok.